src/python/pyproject.toml

```diff
@@ -2,13 +2,44 @@
 requires = [
     "setuptools",
     "wheel",
+    "petab",
 ]
 build-backend = "setuptools.build_meta"
 
-[tool.black]
+
+[project]
+name = "benchmark_models_petab"
+description = "A collection of models with experimental data in the PEtab format"
+dynamic = ["version"]
+readme = { file = "README.md", content-type = "text/markdown" }
+requires-python = ">=3.10"
+license = { file = "LICENSE" }
+authors = [
+    { name = "The Benchmark-Models-PEtab maintainers", email = "yannik.schaelte@gmail.com" }
```

Comment:
You're welcome to add yourself here, as you like.

Reply:
Updated maintainers/authors.